### PR TITLE
issue #852, issue #547, typos

### PR DESCRIPTION
**Pull request by airelil**
List of all fixes:
- issue #852: re-work Application.cpu_usage for Linux platform
- typo in ATSPI `ComboBoxWrapper.text` doc string
- move `client_to_screen` implementation into BaseWrapper. This is to have a single method for all backends
- spaces and indentation in linux/application.py
- issue #547: throw same exception from HwndWrapper.close and UIAWrapper.close
    Both backends should raise the same WindowNotFoundError exception if their `close` methods fail to find the window to close

### CODE REVIEW

File: pywinauto/controls/uiawrapper.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
 import comtypes
 
 from .. import backend
+from ..findwindows import WindowNotFoundError
```
Comment (vasily-v-ryabov) [inline pywinauto/controls/uiawrapper.py]: Maybe it's worth using `from .. import WindowNotFoundError` to align with hwndwrapper.py. Not critical. Just to prepare for removing it from `findwindows`.

File: pywinauto/linux/application.py [hunk 2]
```diff
@@ -120,13 +121,33 @@ def cpu_usage(self, interval=None):
...
+                interval = Timings.cpu_usage_interval
+            time.sleep(interval)
+            after = read_cpu_info()
+            pid_time = (after[0] - before[0]) + (after[1] - before[1])
+            sys_time = (after[2] - before[2]) + (after[3] - before[3])
+            if not sys_time:
+                res = 0.0
+            else:
+                res = 100.0 * (float(pid_time) / float(sys_time))
+            #print("linux app cpu usage: ", res)
```
Comment (vasily-v-ryabov) [inline pywinauto/linux/application.py]: Debug print?

File: pywinauto/linux/application.py [hunk 3]
```diff
@@ -120,13 +121,33 @@ def cpu_usage(self, interval=None):
...
+            if not interval:
+                interval = Timings.cpu_usage_interval
+            time.sleep(interval)
+            after = read_cpu_info()
+            pid_time = (after[0] - before[0]) + (after[1] - before[1])
+            sys_time = (after[2] - before[2]) + (after[3] - before[3])
+            if not sys_time:
+                res = 0.0
+            else:
+                res = 100.0 * (float(pid_time) / float(sys_time))
```
Comment (vasily-v-ryabov) [inline pywinauto/linux/application.py]: Maybe I misunderstand something... In my mind `sys_time` is how much time the app spent in the Linux kernel (in dependent kernel modules) and `pid_time` is how much time the app spent in user space modules. So the formula should be a sum of `sys_time` and `pid_time` divided by `interval`. Nope?

Comment (airelil) [reply pywinauto/linux/application.py]: Yes, you got confused with the variable name. Pls, have a look at `read_cpu_info` helper. It returns four values: user time and kernel times for pid and for all system. So the `pid_time` keeps user and kernel time of the process, while `sys_time` stores user time and kernel time of the system. I should rename it to `total_time`

Comment (vasily-v-ryabov) [reply pywinauto/linux/application.py]: OK, now got it.

File: pywinauto/controls/uiawrapper.py [hunk 4]
```diff
@@ -131,7 +131,7 @@ def __get__(self, obj, cls):
         if obj is None:
             return None
         value = self.fget(obj)
-        setattr(obj, self.func_name, value)
+        #setattr(obj, self.func_name, value)
```
Comment (vasily-v-ryabov) [inline pywinauto/controls/uiawrapper.py]: What is the reason to comment this line?

Comment (airelil) [reply pywinauto/controls/uiawrapper.py]: It was internal debug accidentally committed.

**Pull request merged**